validate.js

```diff
@@ -0,0 +1,13 @@
+const submitButton = document.getElementById('submit');
+const email = document.getElementById('mail');
+
+submitButton.addEventListener('click', () => {
+  const emailString = email.value;
+
+  if (emailString === emailString.toLowerCase()) {
+    submitButton.setCustomValidity('');
+  } else {
+    submitButton.setCustomValidity('Email input characters must be lower case!');
+    submitButton.preventDefault();
```

Comment:
- [ ] **OPTIONAL**
According to the project requirements, you should dynamically render an error message that appears next to the get in touch button and style it as the color / pattern of the website.

_I'm sorry this point didn't show up in the earlier review._